app/code/core/Mage/Core/Model/Store.php

```diff
@@ -364,7 +364,7 @@ public function getConfig($path)
     public function initConfigCache()
     {
         /**
-         * Funtionality related with config separation
+         * Functionally related with config separation
```

Comment:
```suggestion
         * Functionality related with config separation
```

I guess we'll never know the original idea of the author :-)